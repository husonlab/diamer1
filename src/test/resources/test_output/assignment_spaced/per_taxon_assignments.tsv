node id	rank	label	kmers in database	kmer count	kmer count (accumulated)	OVO (ratio: 0.200000) read count	OVO (ratio: 0.200000) read count normalized	OVO (ratio: 0.200000) read count (accumulated)	OVO (ratio: 0.200000) read count normalized (accumulated)	OVO (ratio: 0.800000) read count	OVO (ratio: 0.800000) read count normalized	OVO (ratio: 0.800000) read count (accumulated)	OVO (ratio: 0.800000) read count normalized (accumulated)
0	no rank	root	0	0	86	0	0	2	2	0	0	2	2
1	1 from root	rank1	162	13	13	1	1	1	1	1	1	1	1
2	1 from root	rank2	273	73	73	1	1	1	1	1	1	1	1
4	2 from root	organism2	438	0	0	0	0	0	0	0	0	0	0
5	2 from root	organism3	323	0	0	0	0	0	0	0	0	0	0
3	2 from root	organism1	87	0	0	0	0	0	0	0	0	0	0
