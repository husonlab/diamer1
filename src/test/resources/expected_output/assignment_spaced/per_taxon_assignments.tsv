node id	rank	label	kmers in database	kmer count	kmer count (cumulative)	OVO (0.20) kmer count	OVO (0.20) kmer count (cumulative)	OVO (0.20) norm. kmer count	OVO (0.20) norm. kmer count (cumulative)	OVO (0.50) kmer count	OVO (0.50) kmer count (cumulative)	OVO (0.50) norm. kmer count	OVO (0.50) norm. kmer count (cumulative)	OVO (0.60) kmer count	OVO (0.60) kmer count (cumulative)	OVO (0.60) norm. kmer count	OVO (0.60) norm. kmer count (cumulative)	OVO (0.80) kmer count	OVO (0.80) kmer count (cumulative)	OVO (0.80) norm. kmer count	OVO (0.80) norm. kmer count (cumulative)	OVO (0.90) kmer count	OVO (0.90) kmer count (cumulative)	OVO (0.90) norm. kmer count	OVO (0.90) norm. kmer count (cumulative)	OVO (1.00) kmer count	OVO (1.00) kmer count (cumulative)	OVO (1.00) norm. kmer count	OVO (1.00) norm. kmer count (cumulative)	OVA (0.70) kmer count	OVA (0.70) kmer count (cumulative)	OVA (0.70) norm. kmer count	OVA (0.70) norm. kmer count (cumulative)	norm. kmer count
0	no rank	root	0	0	86	0	2	0	2	0	2	0	2	0	2	0	2	0	2	0	2	0	2	0	2	0	2	0	2	0	2	0	2	0.0
1	1 from root	rank1	162	13	13	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	0.08024691358024691
2	1 from root	rank2	260	73	73	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	0.28076923076923077
4	2 from root	organism2	438	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0
5	2 from root	organism3	323	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0
3	2 from root	organism1	87	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0
